### PR TITLE
fix(clerk-js): Display ellipsis properly where needed

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

### CODE REVIEW

**Comment by desiprisg** [comment]
Think I got them all. Feel free to review.

File: packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx [hunk 1]
```diff
@@ -91,25 +93,32 @@ export const ArrowBlockButton = (props: ArrowBlockButtonProps) => {
...
       )}
-      <Text
-        elementDescriptor={textElementDescriptor}
-        elementId={textElementId}
-        as='span'
-        truncate
-        colorScheme='inherit'
-        variant='buttonSmallRegular'
-        localizationKey={textLocalizationKey}
+      <Flex
```
Comment (nikosdouvlis) [inline packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx:96]: Hey George, this PR looks great. I'd like to check whether this extra div here introduces any layout shifting when the button moves in and out of the loading state, then we can merge

**Review by SokratisVidros — APPROVED** [review]

File: packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx (hunk 1 above)
Comment (nikosdouvlis) [reply packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx:96]: Looks like adding a new div messes with the flex layout of the button, please see the screenshot below: 
image

Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx:96]: Should be fixed. Take a look.

**Review by SokratisVidros — APPROVED** [review]

**Review by nikosdouvlis — APPROVED** [review]

**Pull request merged**